lib/normalizers/active_record/sql.rb

```diff
@@ -19,6 +19,7 @@ def normalize(record)
       private
 
       def cleanup(query)
+        query.gsub!(", ?", "")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.